Changes

```diff
@@ -92,6 +92,9 @@ Working version
 - #8716: Optimize [Array.fill] and [Hashtbl.clear] with a new runtime primitive
   (Alain Frisch, review by David Allsopp, Stephen Dolan and Damien Doligez)
 
+- #8760: `List.concat_map : ('a -> 'b list) -> 'a list -> 'b list`
```

Comment:
Since the PR looks perfect I'm just going to say that usually the types of added symbols are not mentioned in `Changes`. The pattern rather seems to be "Add `List.concat_map`."